recipes/pycoast/meta.yaml

```diff
@@ -0,0 +1,60 @@
+{% set name = "pycoast" %}
+{% set version = "1.1.0" %}
+{% set file_ext = "tar.gz" %}
+{% set hash_type = "sha256" %}
+{% set hash_value = "3359b0cc977b13b4b96628a7c2884e85548a08e57220d78734632f914ed1246c" %}
+
+package:
+  name: '{{ name|lower }}'
+  version: '{{ version }}'
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.{{ file_ext }}
+  '{{ hash_type }}': '{{ hash_value }}'
+
+build:
+  number: 0
+  noarch: python
+  script: python -m pip install --no-deps --ignore-installed .
+
+requirements:
+  host:
```

Comment:
Are any of the dependencies outside of `python`, `pip` and `setuptools` needed to build the package? A quick glance at the source code in `setup.py` looks like the others might not be needed in the `host` section.